### ISSUE TITLE
Non-Accessible Color Schemes Should Be Clearly Labeled

### ISSUE THREAD

**Issue by cgrymala** [WordPress/twentysixteen#180]
In order to comply with the Theme Review accessibility guidelines, the non-accessible color schemes should be clearly labeled as such:

> Alternative themes should be clearly labeled if they do not meet accessibility guidelines.
> https://make.wordpress.org/themes/handbook/review/accessibility/required/#contrasts

I am preparing a Pull Request to address this. Thank you.

**Comment by karmatosed** [comment]
As this PR was closed and the conversation over there, closing this also.
